package.json

```diff
@@ -21,10 +21,12 @@
     "test": "yarn test:backend && yarn test:frontend",
     "test:backend": "cd backend && yarn test",
     "test:frontend": "cd frontend && yarn test",
+    "coveralls": "yarn test --coverage && cat ./backend/coverage/lcov.info | coveralls",
```

Comment:
Using coveralls webhooks, maybe https://github.com/lemurheavy/coveralls-public/issues/544

Why not run tests and coverage for both at the same time?

Reply:
The test script run in Travis is `"yarn test:backend && yarn test:frontend"`, so it actually runs for both but not at the same time because the methods to test the backend and frontend are different, even though both use `jest`.

In the end this creates two coverage reports, one inside frontend and one inside backend, and the `coveralls` script you added point only to the backend.

But yeah, apparently webhooks could do the trick. Do you think this is something easy enough to be part of this PR or should we open a new issue for merging the reports with webhooks?

Reply:
@tristola any opinion so we could close this one?

Reply:
I haven't used this feature, so i need to test that properly - trying to find time at some point to test that

Reply:
I've been doing some big changes to this repo, so I'll merge this PR so I can keep track of the progress in the this branch, which will be deprecated soon!

Thanks for your contribution, soon I'll move your solution to the new master branch.